favicon/faviconData.json

```diff
@@ -0,0 +1 @@
f2fe1ebc0e0d4cdf19e/package_files/favicon-96x96.png","https://realfavicongenerator.net/files/14c8a8334ef6e75031f5bf2fe1ebc0e0d4cdf19e/package_files/favicon.ico","https://realfavicongenerator.net/files/14c8a8334ef6e75031f5bf2fe1ebc0e0d4cdf19e/package_files/manifest.json","https://realfavicongenerator.net/files/14c8a8334ef6e75031f5bf2fe1ebc0e0d4cdf19e/package_files/mstile-144x144.png","https://realfavicongenerator.net/files/14c8a8334ef6e75031f5bf2fe1ebc0e0d4cdf19e/package_files/mstile-150x150.png","https://realfavicongenerator.net/files/14c8a8334ef6e75031f5bf2fe1ebc0e0d4cdf19e/package_files/mstile-310x150.png","https://realfavicongenerator.net/files/14c8a8334ef6e75031f5bf2fe1ebc0e0d4cdf19e/package_files/mstile-310x310.png","https://realfavicongenerator.net/files/14c8a8334ef6e75031f5bf2fe1ebc0e0d4cdf19e/package_files/mstile-70x70.png","https://realfavicongenerator.net/files/14c8a8334ef6e75031f5bf2fe1ebc0e0d4cdf19e/package_files/safari-pinned-tab.svg"],"html_code":"<link rel=\"apple-touch-icon\" sizes=\"57x57\" href=\"/assets/icons/apple-touch-icon-57x57.png\">\n<link rel=\"apple-touch-icon\" sizes=\"60x60\" href=\"/assets/icons/apple-touch-icon-60x60.png\">\n<link rel=\"apple-touch-icon\" sizes=\"72x72\" href=\"/assets/icons/apple-touch-icon-72x72.png\">\n<link rel=\"apple-touch-icon\" sizes=\"76x76\" href=\"/assets/icons/apple-touch-icon-76x76.png\">\n<link rel=\"apple-touch-icon\" sizes=\"114x114\" href=\"/assets/icons/apple-touch-icon-114x114.png\">\n<link rel=\"apple-touch-icon\" sizes=\"120x120\" href=\"/assets/icons/apple-touch-icon-120x120.png\">\n<link rel=\"apple-touch-icon\" sizes=\"144x144\" href=\"/assets/icons/apple-touch-icon-144x144.png\">\n<link rel=\"apple-touch-icon\" sizes=\"152x152\" href=\"/assets/icons/apple-touch-icon-152x152.png\">\n<link rel=\"apple-touch-icon\" sizes=\"180x180\" href=\"/assets/icons/apple-touch-icon-180x180.png\">\n<link rel=\"icon\" type=\"image/png\" href=\"/assets/icons/favicon-32x32.png\" sizes=\"32x32\">\n<link rel=\"icon\" type=\"image/png\" href=\"/assets/icons/android-chrome-192x192.png\" sizes=\"192x192\">\n<link rel=\"icon\" type=\"image/png\" href=\"/assets/icons/favicon-96x96.png\" sizes=\"96x96\">\n<link rel=\"icon\" type=\"image/png\" href=\"/assets/icons/favicon-16x16.png\" sizes=\"16x16\">\n<link rel=\"manifest\" href=\"/assets/icons/manifest.json\">\n<link rel=\"mask-icon\" href=\"/assets/icons/safari-pinned-tab.svg\" color=\"#5bbad5\">\n<link rel=\"shortcut icon\" href=\"/assets/icons/favicon.ico\">\n<meta name=\"msapplication-TileColor\" content=\"#2d89ef\">\n<meta name=\"msapplication-TileImage\" content=\"/assets/icons/mstile-144x144.png\">\n<meta name=\"msapplication-config\" content=\"/assets/icons/browserconfig.xml\">\n<meta name=\"theme-color\" content=\"#ffffff\">","compression":"false"},"files_location":{"type":"path","path":"/assets/icons"},"preview_picture_url":"https://realfavicongenerator.net/files/14c8a8334ef6e75031f5bf2fe1ebc0e0d4cdf19e/favicon_preview.png","version":"0.12"}
```

Comment:
I think this doesn't belong to repo. Could be generated in deploy task I suppose. Correct me if I am wrong.